agent/main.go

```diff
@@ -149,15 +150,16 @@ func main() {
 		logger.Error(err, "could not determine hostname")
 		return
 	}
-
+	_, err = os.Stat(registration.GetBYOHConfigPath())
```

Comment:
Should we also check for err?

Reply:
We cannot exit if there are any errors in this. Do you mean to log it or something else? We are checking errors.Is(err, os.ErrNotExist) for the bootstrap kubeconfig flow

Reply:
> We are checking errors.Is(err, os.ErrNotExist) for the bootstrap kubeconfig flow

This is fine

> Do you mean to log it or something else?

Yes, if `os.State` fails we should at least log an error?

Reply:
Logs don't seem very nice
```
E0704 07:00:33.307968  517472 main.go:155]  "msg"="could not lookup kubeconfig file" "error"="stat /home/kokoni/.byoh/config: no such file or directory"  
I0704 07:00:33.308059  517472 main.go:236]  "msg"="initiated bootstrap kubeconfig flow" 
I0704 07:00:33.327298  517472 csr.go:78]  "msg"="CSR request created"  
I0704 07:00:33.327327  517472 csr.go:82]  "msg"="waiting for client certificate to be issued" 
```